viewer/lib/helpers/title.dart

```diff
@@ -16,10 +16,23 @@ const pathLength = 3;
 
 // title converts a path into a string suitable for display in the app bar.
 String title(String path) {
+  path = path.replaceAll("/locations/global", "");
```

Comment:
"locations/global" was cluttering the display. We may want to do more in the future to avoid confusion about this.